Sergio/HW3/errout/spill.15504902.0.error

```diff
@@ -0,0 +1,6 @@
+Error in file(file, "rt") : cannot open the connection
+Calls: read.csv -> read.table -> file
+In addition: Warning message:
+In file(file, "rt") :
+  cannot open file 'data/spills_1950_1959.csv': No such file or directory
```

Comment:
@Serge757, All of your jobs failed as you can see in the error messages. The reason is I think you have a path like data/spills_1950_1959.csv in the file read side and you are not transferring the data folder..At the execute point only the file exists not the directory